cmake/LibraryConfigurations.cmake

```diff
@@ -59,7 +59,7 @@ if (${PLATFORM} MATCHES "Desktop")
 elseif (${PLATFORM} MATCHES "Web")
     set(PLATFORM_CPP "PLATFORM_WEB")
     set(GRAPHICS "GRAPHICS_API_OPENGL_ES2")
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -s USE_GLFW=3 -s ASSERTIONS=1 --profiling")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} --profiling")
```

Comment:
I was wondering what this did:
https://emscripten.org/docs/tools_reference/emcc.html?highlight=profiling
I think users can control debug builds on their own, by passing this option manually, or perhaps `-DCMAKE_BUILD_TYPE=Debug`
Is there a reason to preserve it still I don't know about?
Perhaps this should just be removed completely?

Reply:
> I think users can control debug builds on their own, by passing this option manually

Agreed.

I got no good reason to keep it. Should I remove it then?

Reply:
Please do.